docs/osf/addons.rst

```diff
@@ -78,13 +276,13 @@ Use the ``NodeLog`` class's named constants when possible,
 Every log action requires a template in ``youraddon/templates/log_templates.mako``. Each template's id corresponds to the name of the log action.
 
 
-Static files for addons
+Static files for add-ons
 ***********************
```

Comment:
Make sure underline lengths are correct, else we get a bunch of warnings when building the docs.

Useful vim tip:

```
yypV*
```